### DISCUSSION TITLE
Forward and back buttons

### DISCUSSION THREAD

**Discussion by kaisalmen** [CodinGame/monaco-vscode-api#528] [Q&A]
@CGNonofr which services are required to make the navigation buttons to work? In the demo the work with views and workbench.

**Comment by CGNonofr** [comment]
I have the feeling you aleady asked the exact same thing ^^ => https://github.com/CodinGame/monaco-vscode-api/discussions/425

Am I wrong?

**Reply by kaisalmen** [reply]
Your memory is better than mine. 😅 

Then, something else is still missing, because the views-service is used and the common one is loaded along with it?
https://typefox.github.io/monaco-languageclient/ghp_appPlayground.html

**Reply by CGNonofr** [reply]
Ok it looks like the HistoryService is not loader: run the `Go Previous` command and it will load it then everything will work

**Reply by CGNonofr** [reply]
You don't use any other service depending on it, like "timeline", "dialog", "search", "chat" or "debug"

can you please create an issue to force this service loading?

**Reply by kaisalmen** [reply]
Done, see https://github.com/CodinGame/monaco-vscode-api/issues/529
